examples/artifact_benchmarking/src/benchmark.py

```diff
@@ -61,7 +62,10 @@ def precision(model: Module, test_dataloader: DataLoader, padding_id: int = -100
         {"class_label": "I-MISC"},
     ),
     tags=("metric", "per-class"),
-)
+)()
```

Comment:
This last call `()` was missing in previous implementations and was what broke the collection in the version previously.

Reply:
Ah yes, because we removed the no-parentheses fastpath for `parametrize` and product.